run/testing/events_storage.e2e_test.go

```diff
@@ -24,10 +24,11 @@ import (
 	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
 )
 
-func TestPubSubEventsService(t *testing.T) {
+func TestPubSubStorageService(t *testing.T) {
 	tc := testutil.EndToEndTest(t)
 
-	service := cloudrunci.NewService("events-storage", tc.ProjectID)
+	service := cloudrunci.NewService("events_storage", tc.ProjectID)
```

Comment:
I think this needs to be a dash, not an underscore?

Reply:
Nice catch!